### ISSUE TITLE
KPGoogleSybc no longer works.

### ISSUE THREAD

**Issue by mpapini64** [walterpg/google-drive-sync#83]
Since this morning, this plugin has stopped working on all my PCs at the same time.
What tells me that the app is blocked after I enter my account email
I contacted Google support (I have Google One) and they told me that if Google has decided to block the app it is because it is violating its standards.
So this plugin is no longer in compliance with the standards?

**To Reproduce**
- I entered the email in the configurator
- the Google Sign-in form opened
- the browser window opened to choose which email to enter with.
- entered the email it tells me
This application is blocked

**Expected behavior**
I expect it to ask me for my password and ask me for confirmation on my mobile phone and finally ask me to confirm that I can access my files.

**Screenshots**
![image](https://github.com/user-attachments/assets/3002cbad-d49b-4738-a5b7-409dd5da629b)

**Please provide the following information:**
-  Edizione	Windows 11 Pro       Versione	23H2    Data installazione:	‎19/‎01/‎2024    Build sistema operativo	22631.4460
Esperienza	Windows Feature Experience Pack 1000.22700.1047.0

 - Chrome Versione 131.0.6778.70 (Build ufficiale) (a 64 bit)
 - Plugin Version  4.1.0
 - KeePass Version 2.57.1
 - "Normal" KP installation
 - ![image](https://github.com/user-attachments/assets/72abcf54-6c56-4b39-9b00-5f7cb580cb6e)

**Comment by Unlifate** [comment]
I also have this issue, but this solution works as a workaround: 
@liteCarma in https://github.com/walterpg/google-drive-sync/issues/79#issuecomment-2275199540

**Comment by 0belius** [comment]
Actually even [this](https://github.com/walterpg/google-drive-sync/issues/79#issuecomment-2295346518) is enough

**Comment by mpapini64** [comment]
The solutions you (@Unlifate and @StephanFA) propose necessarily require the generation of the client and secret id, which I do not have, furthermore I doubt that I would solve this inconvenience, Google considers the app or rather the plugin not safe, at least that's what Google One support told me.
Anyway I found a valid replacement for this plugin, I made three triggers, the first that synchronizes the current DB with a synchronized copy on the Cloud, when a save is made, the second that synchronizes when the DB is opened and finally a third that adds a button and allows you to synchronize manually.
In this way I replaced all the plugin's features.
Obviously on the mobile, my KeePass points to the DB on the cloud directly, since the app does not have the triggers engine.
Anyway if the developer provides me with a solution, then welcome.

**Comment by generalmimon** [comment]
@mpapini64:

> The solutions you (@Unlifate and @StephanFA) propose necessarily require the generation of the client and secret id, which I do not have,

No, you're mistaken, https://github.com/walterpg/google-drive-sync/issues/79#issuecomment-2295346518 suggested by @StephanFA does not. Look at the screenshot in https://github.com/walterpg/google-drive-sync/issues/79#issuecomment-2295346518 carefully - the **Client ID** and **Client Secret** input fields are disabled (grayed out), so you don't have to and in fact can't enter anything there. Apparently, they are disabled as long as you leave the **Use KeePass Google Sync 3.0 App Credentials** checkbox checked.

The workaround in https://github.com/walterpg/google-drive-sync/issues/79#issuecomment-2295346518 works for me and is very easy to do - I literally just ticked two checkboxes and everything works like clockwork again.

**Comment by wenz** [comment]
I can confirm what @generalmimon wrote in https://github.com/walterpg/google-drive-sync/issues/83#issuecomment-2479939814. Works like a charm (at least for now)!

**Comment by golf247** [comment]
I tried #79 but it did not work for me.  It still wanted new credentials.   My work around for the time being was to install drive for desktop and setup Triggers for OnOpen and OnSave to sync to the local G drive version.   Not as nice as KPSYNC and would require G drive on every PC.   Some PCs had a limited access DB that still sync'd to my Gdrive using KPSYNC, thus hoping there is a way to fix this.   

Walter I will say thank you for past and any future efforts.   I suspect google is getting tougher to stay logged in with 'key' type of credentials, vs having to 2FA every time.

**Comment by Robopwner** [comment]
I had this issue as well.
I somehow had the Legacy OAuth2.0  ID & Secret, and once those were cleared and use KeePass Google Sync 3.0 App Credentials and Use Limited Driver Access checked, I was able to sign-in and use KPSync again.
This issue should be looked into.

**Comment by sudeshchouhan** [comment]
Thanks a lot @generalmimon, it works.

**Comment by dregad** [comment]
Duplicate of #79
